man/man1/oci-image-migrate-import.1

```diff
@@ -1,5 +1,5 @@
 .\" Process this file with
-.\" groff -man -Tascii oci-image-migrate.1
+.\" groff -man -Tascii oci-image-migrate-import.1
 .\"
 .\" Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
 .\"
```

Comment:
You need to include the full license piece into the file, ie, these lines are missing:

Licensed under the Universal Permissive License v 1.0 as shown at
 http://oss.oracle.com/licenses/upl.